src/graph_compute/common.h

```diff
@@ -85,6 +89,13 @@
   #ifndef NUMA_INIT
     #define NUMA_INIT 1
   #endif
+  #ifndef NUMA_STEAL
+    #define NUMA_STEAL 1
```

Comment:
I don't like the fact that NUMA_STEAL has different defaults in different places...